packages/peaclock/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/octobanana/peaclock
+TERMUX_PKG_DESCRIPTION="A responsive and customizable clock, timer, and stopwatch for the terminal."
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@efreak"
+TERMUX_PKG_VERSION=0.4.3
+TERMUX_PKG_SRCURL=https://github.com/octobanana/peaclock/archive/0.4.3.tar.gz
```

Comment:
Yeah, sorry, I was a bit unclear, please use `$TERMUX_PKG_VERSION` instead of 0.4.3, but keep https://github.com/octobanana/peaclock (that's how other packages does it)